SPECS/python-cached_property/python-cached_property.spec

```diff
@@ -0,0 +1,184 @@
+%global projectname cached-property
+%global modulename  cached_property
+Summary:        A cached-property for decorating methods in Python classes
+Name:           python-%{modulename}
+Version:        1.5.2
+Release:        9%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/pydanny/%{projectname}
+Source0:        https://github.com/pydanny/%{projectname}/archive/%{version}/%{projectname}-%{version}.tar.gz
+# Disable a couple of test checks that fail with freezegun 0.3.11
+# See https://github.com/pydanny/cached-property/issues/131
+Patch0:         0001-Disable-checks-broken-with-freezegun-0.3.11-131.patch
+# Fix for removal of asyncio.coroutine in Python 3.11
+# https://github.com/pydanny/cached-property/pull/267
+Patch1:         267.patch
+BuildArch:      noarch
+
+%description
+cached_property allows properties in Python classes to be cached until the cache
+is invalidated or expired.
+
+%package -n python%{python3_pkgversion}-%{modulename}
+%{?python_provide:%python_provide python%{python3_pkgversion}-%{modulename}}
+Summary:        A cached-property for decorating methods in Python classes.
+BuildRequires:  python%{python3_pkgversion}-dateutil
+BuildRequires:  python%{python3_pkgversion}-devel
+BuildRequires:  python%{python3_pkgversion}-setuptools
+Provides:       python%{python3_pkgversion}-%{projectname} = %{version}-%{release}
+%if %{with_check}
+BuildRequires:  python3-pip
+%endif
+
+%description -n python%{python3_pkgversion}-%{modulename}
+cached_property allows properties in Python classes to be cached until the cache
+is invalidated or expired.
+
+%prep
+%autosetup -p1 -n %{projectname}-%{version}
+
+%build
+%{__python3} setup.py build
+
+%install
+%{__python3} setup.py install -O1 --skip-build --root %{buildroot}
+
+%check
+pip3 install pytest freezegun
```

Comment:
We've had issues with `pytest` updates breaking tests in the past. Please take a look at the test output from the logs and freeze the version of `pytest` to whatever was used now.